tests/unit/cmd/test_config.py

```diff
@@ -45,7 +50,7 @@
 
 
 class TestConfigList:
-    def test_valid_list(self, cli_fs_runner):
+    def test_valid_list(self, cli_fs_runner, config_list_json_schema):
```

Comment:
I would prefer if there was two separate tests: one for text and one for the JSON output. This way we have better diagnostics if one of them fails (and each test is shorter).

You can put the setup in a class-level fixture to avoid code duplication.